### ISSUE TITLE
enhancement: split import statements on same line

### ISSUE THREAD

**Issue by brandon1024** [brandon1024/java-sort-imports.vim#5]
If there are import statements on the same line (see example below), the plugin will skip over it. We should make an effort to reformat such lines.

```
import java.util.ArrayList;import java.util.Collections;import java.util.HashMap;
```

**Comment by brandon1024** [comment]
Be sure to update the `limitations` section in the help doc page.

**Comment by brandon1024** [comment]
To be fixed in #19
